harmonica/gravity_corrections.py

```diff
@@ -69,3 +74,72 @@ def bouguer_correction(topography, density_crust=2670, density_water=1040):
     density[oceans] = -1 * (density_water - density_crust)
     bouguer = 1e5 * 2 * np.pi * GRAVITATIONAL_CONST * density * topography
     return bouguer
+
+def atmospheric_correction(topography):
+    """
+    Calculates atmospheric correction as per Hinze 2005 eqn 5.
+    
+    .. math::
+
+        g_{atm} = 0.874 - 9.9E-5 h + 3.56E-9 h^2 
+
+    in which :math:`h` is the elevation of the station above the ellipsoid
+    and :math:`g_{atm}` is the gravitational effect of the atmospheric the air
+    mass in mGal.
+
+    Parameters
+    ----------
+    topography : array or :class:`xarray.DataArray`
+        Topography height and bathymetry depth in meters.
+        Should be referenced to the ellipsoid (ie, geometric heights).
+        
+
+    Returns
+    -------
+    atmos_correction : array or :class:`xarray.DataArray`
+    The gravitational effect of the atmosphere in mGal
+
+    """
+    atmos_correction = 0.874 - 9.9e-05 * topography + 3.56e-09 * topography**2
+    
+    return atmos_correction
+
+
+def spherical_bouguer_cap_correction(topography):
```

Comment:
Does the density not enter into this equation?

Reply:
I think that formulation assumes a 2670 density, but see comments above about spherical bouguer correction which would take care of this

Reply:
See comments above about using the full analytic expression.  This approximation assumes 2670, so need to scale to other density.